src/lib/d3/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2010-2015, Michael Bostock
```

Comment:
A good idea. Created a jira for it: https://issues.fluidproject.org/browse/FLOE-405